examples/ut_custom_reporter.tps

```diff
@@ -0,0 +1,21 @@
+create or replace type ut_custom_reporter under ut_dbms_output_suite_reporter
+(
+-- Author  : PAZZZ
+-- Created : 20.07.2016 22:51:51
+-- Purpose : 
+
+-- Attributes
```

Comment:
May I ask you to delete this comments generated by IDE? If I do this it will brake your PR with conflict.